src/app/codegen-data-model-provider/CodegenDataModelProvider.cpp

```diff
@@ -31,6 +34,69 @@ namespace chip {
 namespace app {
 namespace {
 
+/// Handles going through callback-based enumeration of generated/accepted commands
+/// for CommandHandler interface based items.
+///
+/// Offers the ability to focus on some operation for finding a given
+/// command id:
+///   - FindFirst will return the first found element
+///   - FindExact finds the element with the given id
+///   - FindNext finds the element following the given id
+class EnumeratorCommandFinder
+{
+public:
+    enum class Operation
+    {
+        FindFirst, // Find the first value in the list
+        FindExact, // Find the given value
```

Comment:
Is `FindExact` planned to be used in the future or is it no longer needed?

Reply:
FindExact is used to query by path - it is a query of "does this exist". I failed to update the `GetAcceptedCommandInfo` with the new class though ... will update it.

Reply:
Updated GetAcceptedCommandInfo to use this.